sdk/nodejs/ec2/securityGroup.ts

```diff
@@ -176,8 +176,8 @@ import * as utilities from "../utilities";
  *         input: exampleAwsVpcEndpoint.securityGroupIds,
  *     }).then(invoke => invoke.result),
  * }});
- * const exampleResourceProvisioner0 = new command.local.Command("exampleResourceProvisioner0", {create: `            aws ec2 modify-vpc-endpoint --vpc-endpoint-id ${exampleAwsVpcEndpoint.id} --remove-security-group-ids ${_default.id}
- * `}, {
+ * const exampleResourceProvisioner0 = new command.local.Command("exampleResourceProvisioner0", {create: _default.then(_default => `            aws ec2 modify-vpc-endpoint --vpc-endpoint-id ${exampleAwsVpcEndpoint.id} --remove-security-group-ids ${_default.id}
+ * `)}, {
```

Comment:
This diff looks a bit odd and it appears to be being reset during the CI build. Perhaps something was wrong during your local build?

Reply:
Yeah I am a little confused where this is coming from, I pushed again without this.